src/context/WalletProvider/SelectModal.tsx

```diff
@@ -123,7 +123,7 @@ export const SelectModal = () => {
       <ModalBody>
         <Text mb={6} color='text.subtle' translation={'walletProvider.selectModal.body'} />
         <Grid mb={6} gridTemplateColumns={gridTemplateColumnsProp} gridGap={4}>
-          {adapters &&
```

Comment:
`state.adapters` is now irrelevant. This was the whole reason we loaded the whole hdwallet packages trees, we're now lazy